pkg/tsdb/azuremonitor/azure-resource-graph-datasource.go

```diff
@@ -179,15 +179,38 @@ func (e *AzureResourceGraphDatasource) executeQuery(ctx context.Context, query *
 	if err != nil {
 		return dataResponseErrorWithExecuted(err)
 	}
-	if frame.Meta == nil {
-		frame.Meta = &data.FrameMeta{}
+
+	azurePortalUrl, err := getAzurePortalUrl(dsInfo.Cloud)
+	if err != nil {
+		return dataResponseErrorWithExecuted(err)
 	}
-	frame.Meta.ExecutedQueryString = req.URL.RawQuery
 
-	dataResponse.Frames = data.Frames{frame}
+	url := azurePortalUrl + "/#blade/HubsExtension/ArgQueryBlade/query/" + url.PathEscape(query.InterpolatedQuery)
```

Comment:
this can still be made in a more safe way using [URL built in functions](https://pkg.go.dev/net/url#example-URL). Something like:

```
	u, err := url.Parse(azurePortalUrl)
	if err != nil {
		return dataResponseErrorWithExecuted(err)
	}
	queryParam, err := url.ParseQuery(query.InterpolatedQuery)
	if err != nil {
		return dataResponseErrorWithExecuted(err)
	}
	u.Path = path.Join(u.Path, "/#blade/HubsExtension/ArgQueryBlade/query/")
	q := u.Query()
	q.Set("query", queryParam.Get("query"))
	u.RawQuery = q.Encode()

  	frameWithLink := addConfigData(*frame, u.URL())
```

Reply:
It didn't quite work:
`u.URL()` should be `u.String()`
The generated path is `https://portal.azure.com/%23blade/HubsExtension/ArgQueryBlade/query?query=`
`#` is somehow escaped and query is not parsed.